Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -1513,8 +1518,14 @@ on revIDEInitialiseIDELibrary
    
    ## Initialise property library
    revIDEPropertyLibraryInitialise
+   
+   ideSubscribe "ideExtensionsChanged"
 end revIDEInitialiseIDELibrary
 
+on ideExtensionsChanged
+   revIDERegenerateBuiltDictionaryData
```

Comment:
@livecodeali would it be a good idea to add in a delay and cancel here so if multiple extensions are loaded the docs are regenerated only once?

Reply:
I think it's maybe not much of an issue, as the time to reassemble the main `.js` is pretty tiny, especially compared to the time taken to generate the json from an `api.lcdoc`

Reply:
OK, cool